### ISSUE TITLE
aws.ec2_metrics system test fails

### ISSUE THREAD

**Issue by adriansr** [elastic/integrations#1566]
The system test fails with:
> could not find hits in metrics-aws.ec2_metrics-ep data stream

**Comment by kaiyan-sheng** [comment]
I agree with your assumption and actually this happens quite often. We added a `latency` config option to deal with that in [Beats](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-aws.html#_module_specific_configuration_notes) but I haven't migrated that config option into the aws package yet. Will do that right now!
